app/src/pages/outside/loginPage/serviceVersionsBlock/serviceVersionItem/serviceVersionItem.jsx

```diff
@@ -40,7 +40,7 @@ export class ServiceVersionItem extends Component {
     serviceName: PropTypes.string,
     serviceVersion: PropTypes.string,
     serviceNewVersion: PropTypes.string,
-    isDeprecated: PropTypes.bool,
+    isDeprecated: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
I think that it shouldn't have `string` type.
Please convert the prop value to `Boolean` to avoid misunderstanding in the future.

Reply:
In some cases the string type is passed here

Reply:
This prop is used as a boolean, so strings should be cast to boolean in that case.